### ISSUE TITLE
Specify functions' access modifiers

### ISSUE THREAD

**Issue by dashaluna** [dashaluna/better-font-awesome#7]

**Comment by tcrsavage** [comment]
As a general rule of thumb. Specify public on everything by default, protected on things you don't want developers accessing directly and private on things which should never be messed with.

Protected/Private methods should always sit at the bottom of the class

**Comment by BronsonQuick** [comment]
@tcrsavage Ahh nice tip! I always tended to sort functions alphabetically rather than scope! :+1:

**Comment by dashaluna** [comment]
Yeah, I usually declare functions close to the file's workflow.. Good tip though, I will be looking out for it :)
